code/modules/mob/living/carbon/carbon_stripping.dm

```diff
@@ -10,7 +10,8 @@
 	return get_strippable_alternate_action_internals(get_item(source), source)
 
 /datum/strippable_item/mob_item_slot/back/alternate_action(atom/source, mob/user)
-	return strippable_alternate_action_internals(get_item(source), source, user)
+	if(!..())
```

Comment:
Why is this one `!..()` while the others are `..()`?

Reply:
This is a failure on my part to document on the parent. It currently returns `TRUE` when the alternate action is blocked. It should be changed to return `TRUE` by default and `FALSE` if blocked, which makes more sense.
https://github.com/tgstation/tgstation/blob/be4cd777eefcc472e8b52688e60f78fe0b4cf6c0/code/datums/elements/strippable.dm#L184-L187